userspace/libsinsp/user.cpp

```diff
@@ -190,6 +190,14 @@ scap_userinfo *sinsp_usergroup_manager::add_user(const string &container_id, uin
 
 		usr = &userlist[uid];
 	}
+	else  if (name != NULL)
+	{
+		// Update user if it was already there
+		strlcpy(usr->name, name, MAX_CREDENTIALS_STR_LEN);
```

Comment:
Consider the case where a "uid, NA, NA, NA" user is added because users and groups informations are not available for some reasons. 
If then in a second moment, users and groups info are available, overwrite old ones.